bundle/manifests/gatekeeper-operator.clusterserviceversion.yaml

```diff
@@ -494,5 +494,5 @@ spec:
   maturity: alpha
   provider:
     name: Open Policy Agent Gatekeeper community
-  replaces: gatekeeper-operator.v0.2.1
-  version: 0.3.0-alpha.1
+  # replaces: none
```

Comment:
Is this right? I thought it needed to replace something, for upgrades? Or is that tested/ handled after the candidate...

Mostly worried because this doesn't match what's in `config/manifests/bases/gatekeeper-operator.clusterserviceversion.yaml`, which has
```
replaces: gatekeeper-operator.0.0.0
```
Which also seems strange now that I look at it.